k8s/perftest/common/bsp/bulk-scan-processor.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: bulk-scan-processor
-    version: 0.2.20
+    version: 0.2.18
```

Comment:
did you mean for this to go to 18?

Reply:
should be 19 I think

Reply:
ooh. didn't look at diff. just clicked revert button on pr landing page ;). lemme push extra commit